### PR TITLE
Updating `ReadMe` documentation for Backstage Terraform Integrations

### PR DESCRIPTION
**Pull request by elamaran11**
Updating `ReadMe` documentation for Backstage Terraform Integrations to support package integrations with [Backstage Terraform Integrations](https://github.com/cnoe-io/backstage-terraform-integrations/tree/main) repo.

### CODE REVIEW

**Comment by nimakaviani** [comment]
+1 to what @nabuskey said. 

The idea with integrations is for users to go to the repos for the integrations and extensions to decide and follow next steps

**Comment by elamaran11** [comment]
@nimakaviani @nabuskey Please check my updates based on your comments. I agree to your recommendation, we dont need to update the base `readMe`.

File: examples/terraform-integrations/README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-``` 
-./idpbuilder create --use-path-routing --package-dir examples/ref-implementation --package-dir examples/terraform-integrations 
+```bash
+idpbuilder create \
+  --use-path-routing \
+  --package-dir examples/ref-implementation \
+  --package-dir examples/terraform-integrations
 ```
+
+As you see above, this add-on to `idpbuilder` has a dependency to the [reference implementation][../ref-implementation/]. This stack add-on goes together with the work done under [backstage-terraform-integrations](https://github.com/cnoe-io/backstage-terraform-integrations/). Once the add-on is enabled, the user will need to follow the setup discussed in the [backstage-terraform-integrations](https://github.com/cnoe-io/backstage-terraform-integrations/) repo for the remainder of the configuration, and terraform integrations should work.
```
Comment (nimakaviani) [inline examples/terraform-integrations/README.md:14]:
```suggestion
As you see above, this add-on to `idpbuilder` has a dependency to the [reference implementation](../ref-implementation/). This stack add-on goes together with the work done under [backstage-terraform-integrations](https://github.com/cnoe-io/backstage-terraform-integrations/). Once the add-on is enabled, the user will need to follow the setup discussed in the [backstage-terraform-integrations](https://github.com/cnoe-io/backstage-terraform-integrations/) repo for the remainder of the configuration, and terraform integrations should work.```

File: examples/terraform-integrations/README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
 # Terraform Integrations for Backstage
 
-This is an experimental effort allowing the users of the idpBuilder to run terraform modules using the tooling in place.
+IDP builder is now experimentally extensible to launch custom terraform patterns using package extensions. This is an experimental effort allowing the users of the `idpBuilder` to run terraform modules using the tooling in place.
```
Comment (nimakaviani) [inline examples/terraform-integrations/README.md:3]:
```suggestion
`idpBuilder` is now experimentally extensible to launch custom terraform patterns using package extensions. This is an experimental effort allowing the users of the `idpBuilder` to run terraform modules using the tooling in place.
```

**Review by nimakaviani — APPROVED** [review]
LGTM

**Pull request merged**